coord/src/search_cluster.c

```diff
@@ -408,6 +408,7 @@ MRCommandGenerator SearchCluster_MultiplexCommand(SearchCluster *c, MRCommand *c
       size_t newlen = 0;
       const char *target = lookupAlias(cmd->strs[mux->keyOffset], &newlen);
       if (oldlen != newlen) {
+        // This condition seems odd. Can't they have the same length but different values?
```

Comment:
Did you got a chance to look at it further?

Reply:
Yes, will be fixed in a separate PR